src/grafana_agent.py

```diff
@@ -110,7 +110,7 @@ def __init__(self, *args):
             dest=charm_root.joinpath(*DASHBOARDS_DEST_PATH.split("/")),
         )
 
-        for rules in [self.loki_rules_paths, self.metrics_rules_paths, self.dashboard_paths]:
+        for rules in [self.loki_rules_paths, self.dashboard_paths]:
```

Comment:
Not sure why we had `dashboard_paths` in the list before, but shouldn't we keep `metrics_rules_paths`?

Reply:
No the logic is now handled elsewhere.